# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/fedcm/__dir__.ini
# Commit: d07def1365a4
# Full Hash: d07def1365a49dc8ecbf5b6c10fdfbd5ec750932
# Author: pstanciu <pstanciu@mozilla.com>
# Date: 2025-06-11 18:38:51
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing WPT failures @lfedcm-identity.create-store-collect.tentative.sub.https.html
#   
#   This reverts commit 03d4474e360f28be0fb075379f4e4535bd34b252.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 4e25ddaadc86 -r d07def1365a4 testing/web-platform/meta/fedcm/__dir__.ini
--- a/testing/web-platform/meta/fedcm/__dir__.ini	Wed Jun 11 13:42:13 2025 +0000
+++ b/testing/web-platform/meta/fedcm/__dir__.ini	Wed Jun 11 18:04:27 2025 +0300
@@ -1,4 +1,4 @@
 disabled:
   if os == "android": https://bugzilla.mozilla.org/show_bug.cgi?id=1621476
-prefs: [dom.security.credentialmanagement.identity.enabled:true, dom.security.credentialmanagement.identity.select_first_in_ui_lists:true]
+prefs: [dom.security.credentialmanagement.identity.enabled:true, dom.security.credentialmanagement.identity.lightweight.enabled:true, dom.security.credentialmanagement.chooser.testing.enabled:true]
 leak-threshold: [default:51200]